src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1946,9 +1946,15 @@ internal void FocusCustomNodeWorkspace(Guid symbol)
             }
         }
 
-        internal void ShowElement(NodeModel e)
+        /// <summary>
+        /// Attempts to find a NodeModel and focuses the view around it
+        /// Default boolean introduced to allow for this method to be used in Automatic mode
+        /// </summary>
+        /// <param name="e"></param>
+        /// <param name="automatic"></param>
+        internal void ShowElement(NodeModel e, bool automatic = true)
         {
-            if (HomeSpace.RunSettings.RunType == RunType.Automatic)
+            if (HomeSpace.RunSettings.RunType == RunType.Automatic && automatic)
```

Comment:
I'm afraid, this looks hacky. Why can't we say:
```
if(automatic)
{
HomeSpace.RunSettings.RunType == RunType.Automatic;
return;
}
```

Reply:
Hi! Do you maybe want to have a quick Slack catch-up? These last few things are legacy code for me. I know they are used in TuneUp extension, which is not something I have worked on. Maybe it might be better to judge the extent of this PR - if you think it's a good time to resolve some of these issues, I agree, and let's go over the code and agree on what needs to happen and I will implement it.